cijoe/scripts/qemu_guest_start_custom_nvme.py

```diff
@@ -231,18 +231,18 @@ def namespace(controller_id, nsid, aux={}):
     drives.append(drv_pi3)
 
     # Nvme4n4 - NVM namespace with PI type 1 and extended LBA
-    drv_pi1_ex, qemu_nvme_dev_pi1_ex = namespace(
-        controller_id5, 4, {"ms": 8, "mset": 1, "pi": 1}
-    )
-    nvme += qemu_nvme_dev_pi1_ex
-    drives.append(drv_pi1_ex)
+    #drv_pi1_ex, qemu_nvme_dev_pi1_ex = namespace(
```

Comment:
I would vote to just remove these parts for the final commit, instead of leaving them commented out.

They can always be added back in later if needed.